### ISSUE TITLE
Can't run cloned copy from local machine.

### ISSUE THREAD

**Issue by 80billion** [cynicaloptimist/improved-initiative#309]
Sorry if this is not the appropriate place to post this. I followed the instructions to download and run improved initiative locally. Installed Node.js (LTS), downloaded improved initiative, extracted to local drive, ran NPM Install from that directory using an elevated command prompt. Getting many errors that files or directories don't exist. I've tried it on multiple devices (both windows and linux) and had the same results. I'm new to github so it could be user error, any help would be appreciated.

[2018-12-13T20_30_45_432Z-debug.log](https://github.com/cynicaloptimist/improved-initiative/files/2677810/2018-12-13T20_30_45_432Z-debug.log)

**Comment by cynicaloptimist** [comment]
Not sure how to troubleshoot this, sorry!

**Comment by TheKrush** [comment]
Here's a batch file I use for building on my windows box:

```
title improved-initiative build

cd improved-initiative
npm install
npm audit fix
npm test
pause
cd ..
```

Also do you already have Node.js installed on the machine?

**Comment by 80billion** [comment]
I was able to get this working earlier today. I made a couple of changes so I'm not sure what actually fixed it. Here is what I did just in case someone else runs into this in the future.

1) I downloaded Node.js again but got the regular most up to date version rather than the long term support version. I did have it installed previously but it was LTS.
2) While reinstalling node.js I checked the box to install the additional compiling tools.
3) When I launched github it said there were quite a few commits that hadn't been applied to my clone so I fetched all the updates. I do see the site has been updated since I last looked so I'm guessing that's all that is, I know I had clicked the fetch button previously so I'm rather confident I was using the latest code at that time.
4) I know I had tried it previously but this time I made sure I launched github desktop as an admin so that the command line tool would also run as admin when doing the install command. 

Everything downloaded and installed this time. I will try running the audit fix since it did pop up with a couple of things after the install. Anyway I am now able to launch this locally without internet access. I run games in a shop with no wifi so that's important for me. I'm very much looking forward to using this. Thank you so much for developing this tool and to both of you for commenting.

Thanks again.
